app/services/push_notification/account_reinstated_event.rb

```diff
@@ -0,0 +1,17 @@
+module PushNotification
+  class AccountReinstatedEvent
+    include IssSubEvent
+
+    EVENT_TYPE = 'https://schemas.openid.net/secevent/risc/event-type/account-enabled'.freeze
```

Comment:
let's rename the class too to match the event? `AccountEnabledEvent`? and maybe we can add a class comment that this is used for account reinstatement?

Reply:
sure